src/main/resources/i18n/Resources_zh_Hans.properties

```diff
@@ -23,13 +23,9 @@ preferencesDialogTabPatternsLinkText=查看文档样式格式
 preferencesDialogTabPatternsIgnoreDescription=符合这些文档样式的档案会被忽略
 preferencesDialogIgnoreSectionTitle=忽略这些文档或是文件匣
 preferencesDialogTabMain=主窗口
-#TODO translate - this is a title of a preferences section
-preferencesDialogVariousSectionTitle=Various
-#TODO translate
-preferencesDialogAlbumDelimiterLabel=Delimiter to use between folder names when creating an album name
-#TODO translate - this is a preview of how an album name would look; %s is the album name delimiter, please keep it there; it will be formatted like
-# "Sample album name preview:  Photos: Spain: 2021"
-preferencesDialogAlbumDelimiterExampleLabel=Sample album name preview: Photos%sSpain%s2021
+preferencesDialogVariousSectionTitle=偏好设置种类
+preferencesDialogAlbumDelimiterLabel=相册名称路径分隔符
+preferencesDialogAlbumDelimiterExampleLabel=相册名称预览(请保留路径分隔符%s, 相册名称Photos%sSpain%s2021会被自动生成为Photos：Spain：2021）
```

Comment:
This should just say "Sample album name preview: Photos%sSpain%s2021", I think you translated the comment too by mistake - can you correct please?

Reply:
You can translate "Photos", "Spain" and "2021" too, but please keep the placeholders.